### PR TITLE
My mobile skeleton Portfolio

### PR DESCRIPTION
**Pull request by aeon9300**
I have learned to design a personal portfolio webpage. (Mobile version)

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+  font-family: 'Roboto', sans-serif;
+  font-size: 18px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 24px;
+  text-align: center;
+  margin-left: 10px;
+}
+
+.section-headline {
```
Comment (keneogu) [inline style.css:86]: - [x] Your section-headline should have a display of flex, and the contents should be aligned to the center with margins
Your contents seems to align to the left which is different from the actual design
your design
ay

Figma design
ay1

**Review by keneogu — CHANGES_REQUESTED** [review]
Hi @aeon9300 , 

Good job so far!

- Linters working properly
- Well documented PR description
- Commit messages are meaningful
- Good HTML structure
- Good UI design

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [x] I suggest you make use of the same fonts used in the Figma design template
Your design
ay
Figma design
ay1

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Remove the MIT link on your readme file since is broken

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------

File: style.css (hunk 1 above)
Comment (aeon9300) [reply style.css:86]: Thank you. My 'section-headline' now has a display of flex.

image

Comment (aeon9300) [reply style.css:86]: I have also modified the font to display in the same way as the Figma Template.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        <li class="social-2"><a href="#"><i class="fab fa-linkedin-in fa-lg"></i></a></li>
+        <li class="social-3"><a href="#"><i class="fab fa-angellist fa-lg"></i></a></li>
+        <li class="social-4"><a href="#"><i class="fab fa-twitter fa-lg"></i></a></li>
+        <li class="social-5"><a href="#"><i class="fab fa-medium-m"></i></a></li>
+        </ul>
+
+    </div>
+    
+    <div class="hero-image">
+        <img src="images/background-mobile.svg" alt="" />
```
Comment (ahmetbozaci) [inline index.html:51]: ### OPTIONAL
- [ ] The section has the proper background image (decorations are always background images, not img tags) You can change it for make it better.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+            <nav>
+                <div class="menu1">
+                    <div class="menu"></div>
+                    <div class="menu"></div>
+                    <div class="menu"></div>
+                </div>
+            </nav>
+            </div>
+            
+        </header>
```
Comment (ahmetbozaci) [inline index.html:28]: ### OPTIONAL

- [ ] The header is not like in the original design. You can change it to make it better. (ignore the yellow border in the picture.)
image

___
### ORIGINAL
image

**Review by ahmetbozaci — APPROVED** [review]
# STATUS: APPROVED 🥳 🥳 🥳 

### Hello @aeon9300  ✋🏼 
#### Your project is great. I approve of it. There is nothing else to say other than... it's time to merge it and delete the branch. You can submit this project on your dashboard.

___ 

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
___

### Cheers and Happy coding! 💻 👏
aproved

___

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_


_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mentioning me as [ahmetbozaci](https://github.com/ahmetbozaci), I will receive a notification and will respond._
___
[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Pull request merged**